### DISCUSSION TITLE
I have a posts api that has title,content and icon fields, when I access this api the data returned in the icon field is an id, from that id I call the api Get url/files/:id to get the photo information image , but no path is returned, so how do I dump this image to the frontend?

### DISCUSSION THREAD

**Discussion by Martian95** [directus/directus#19264] [Feature Requests]
### Summary

need the url of image file where they were stored

### Basic Example

ex :url/assets/posts/my-image.png

### Motivation

it help us so much

### Detailed Design

no

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

no

### Alternatives

no

### Adoption Strategy

no

**Comment by DanielBiegler** [comment]
Hey there. Please consult the extensive docs: https://docs.directus.io/reference/files.html

**Reply by Martian95** [reply]
Thanks u somuch i resolved it
